Remove published field from summaries

octopusinvitro opened the pull request:
Published fields for summaries are a bit different from blog posts.
We assume that summaries are always published.

Also, allowing unpublished summaries means we have to add code to
deal with that:

At the moment, in order for `find_or_empty` to work properly, since
it calls `find_all.first`, we should not have summaries with
`published: false`. This is because then `none?` will be `false` but
`find_all` will return an empty array, and `find_all.first` will
return a `nil` instead of a proper document. This will break the app,
as it will throw an error when we try to call `.body` on it.

The fields were removed using:

```bash
perl -i -p -e 's/published: true\n//' summaries/*.md
perl -i -p -e 's/published: false\n//' summaries/*.md
```

mhl reviewed (changes requested):
I guess you ran the commands on an earlier version of the data because it's missed `summaries/gov:mallam-nasir-el-rufai.md`, I'm afraid.

One other small thing is a typo in the commit message: "The fields where removed using:" should be "The fields were removed using:"

octopusinvitro commented:
@mhl feedback addressed :+1:

octopusinvitro committed:
Published fields for summaries are a bit different from blog posts.
We assume that summaries are always published.

Also, allowing unpublished summaries means we have to add code to
deal with that:

At the moment, in order for `find_or_empty` to work properly, since
it calls `find_all.first`, we should not have summaries with
`published: false`. This is because then `none?` will be `false` but
`find_all` will return an empty array, and `find_all.first` will
return a `nil` instead of a proper document. This will break the app,
as it will throw an error when we try to call `.body` on it.

The fields were removed using:

perl -i -p -e 's/published: true\n//' summaries/*.md
perl -i -p -e 's/published: false\n//' summaries/*.md

mhl reviewed (approved):
:+1:
